mitodl/mit-open
# email notification management

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/4891 


### Description (What does it do?)
Adds a notification preferences page for managing email frequency


### How can this be tested?
1. Checkout this branch.
2. login and go to your dashboard. You should see an email notifications tab on the left column.
3. change your preferences and see that it gets saved.

### Additional Context
After a discussion with @Ferdi  and design team - this feature will be revamped in a follow up PR according to https://github.com/mitodl/hq/issues/4891#issuecomment-2248873641 along with another PR that fundamentally changes how the emails are sent out (individual emails vs single digest)
